### PR TITLE
edge-22.10.2

### PR DESCRIPTION
**Pull request by adleong**
This edge release fixes an issue with CNI chaining that was preventing the
Linkerd CNI plugin from working with other CNI plugins such as Cilium. It also
includes several other fixes.

* Updated Grafana dashboards to use variable duration parameter so that they can
  be used when Prometheus has a longer scrape interval (thanks @TarekAS)
* Fixed handling of .conf files in the CNI plugin so that the Linkerd CNI plugin
  can be used alongside other CNI plugins such as Cilium
* Added a `linkerd diagnostics policy` command to inspect Linkerd policy state
* Added a check that clusterip services are in the cluster networks
* Added a noop init container to injected pods when the cni plugin is enabled
  to prevent certain scenarios where a pod can get stuck without an IP address
* Fixed a bug where the proxy-version annotation could be empty

Signed-off-by: Alex Leong <alex@buoyant.io>

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
...
+
+* Updated Grafana dashboards to use variable duration parameter so that they can
+  be used when Prometheus has a longer scrape interval (thanks @TarekAS)
+* Fixed handling of .conf files in the CNI plugin so that the Linkerd CNI plugin
+  can be used alongside other CNI plugins such as Cilium
+* Added a `linkerd diagnostics policy` command to inspect Linkerd policy state
+* Added a check that clusterip services are in the cluster networks
+* Added a noop init container to injected pods when the cni plugin is enabled
+  to prevent certain scenarios where a pod can get stuck without an IP address
+* Fixed a bug where the proxy-version annotation could be empty
```
Comment (hawkw) [inline CHANGES.md:17]: nit/tioli: should this say 
```suggestion
* Fixed a bug where the `config.linkerd.io/proxy-version` annotation could be empty
```

File: CHANGES.md [hunk 2]
```diff
@@ -1,5 +1,21 @@
...
+Linkerd CNI plugin from working with other CNI plugins such as Cilium. It also
+includes several other fixes.
+
+* Updated Grafana dashboards to use variable duration parameter so that they can
+  be used when Prometheus has a longer scrape interval (thanks @TarekAS)
+* Fixed handling of .conf files in the CNI plugin so that the Linkerd CNI plugin
+  can be used alongside other CNI plugins such as Cilium
+* Added a `linkerd diagnostics policy` command to inspect Linkerd policy state
+* Added a check that clusterip services are in the cluster networks
+* Added a noop init container to injected pods when the cni plugin is enabled
```
Comment (hawkw) [inline CHANGES.md:15]: nit:
```suggestion
* Added a noop init container to injected pods when the CNI plugin is enabled
```

File: CHANGES.md [hunk 3]
```diff
@@ -1,5 +1,21 @@
...
+This edge release fixes an issue with CNI chaining that was preventing the
+Linkerd CNI plugin from working with other CNI plugins such as Cilium. It also
+includes several other fixes.
+
+* Updated Grafana dashboards to use variable duration parameter so that they can
+  be used when Prometheus has a longer scrape interval (thanks @TarekAS)
+* Fixed handling of .conf files in the CNI plugin so that the Linkerd CNI plugin
+  can be used alongside other CNI plugins such as Cilium
+* Added a `linkerd diagnostics policy` command to inspect Linkerd policy state
+* Added a check that clusterip services are in the cluster networks
```
Comment (hawkw) [inline CHANGES.md:14]: nit:
```suggestion
* Added a check that ClusterIp services are in the cluster networks
```

**Review by hawkw — COMMENTED** [review]
couple tiny docs nits, but lgtm otherwise!

**Review by kleimkuhler — APPROVED** [review]

**Review by hawkw — APPROVED** [review]

**Pull request merged**